Tests/test_mmtf.py

```diff
@@ -12,13 +12,49 @@
 import warnings
 import os
 import tempfile
+from Bio import PDB
```

Comment:
You can follow the convention below and simply do `from Bio.PDB import read`

Reply:
But isn't that inconsistent with other subpackages like `SeqIO` where always SeqIO.read(..) is used?

Reply:
Well, "read" on its own is a very ambiguous function name so ``SeqIO.read`` or ``AlignIO.read`` etc make much more readable code. I would recommend using ``BDB.read`` in our documentation for the same reason.

However, if the rest of ``test_mmtf.py`` uses the other style, that's fine.

Reply:
👍